pkg/controller/master-controller-manager/usersshkeyssynchronizer/controller_test.go

```diff
@@ -67,15 +67,18 @@ func TestUserSSHKeysClusterRemove(t *testing.T) {
 					"seed_test": fake.NewFakeClient(
 						&kubermaticv1.Cluster{
 							ObjectMeta: metav1.ObjectMeta{
-								DeletionTimestamp: &deletionTimestamp,
 								Name:              "test_cluster_1",
-								Namespace:         "test_namespace",
```

Comment:
Non-namespaced resources *must not* have a namespace set, or creation will fail.